test/new-e2e/system-probe/config/vmconfig.json

```diff
@@ -8,17 +8,37 @@
                 {
                     "dir": "bionic-server-cloudimg-amd64.qcow2",
                     "tag": "bionic",
-                    "image_source": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/bionic-server-cloudimg-amd64.qcow2"
+                    "image_source": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/bionic-server-cloudimg-amd64.qcow2-test-only"
```

Comment:
What's up with the `-test-only` suffix?

Reply:
Comes from [here for example](https://github.com/DataDog/ami-builder/blob/usama.saqib/kernel-version-testing-ami/.gitlab-ci.yml#L139).

I add the suffix when experimenting with different builds, so we don't end up overwriting the stable images used in main.

Here I was experimenting with a leaner build, which removes clang thus reducing the images size by 3Gi.

Reply:
What is the process for promoting images from `test-only`? Should we merge to `main` references to `test-only` images?

Reply:
They should not be merged into `main`. I will remove them once I have the correct images built and pushed to `S3`